### ISSUE TITLE
grunt bower:install doesn't work for polymer

### ISSUE THREAD

**Issue by indiesquidge** [yatskevich/grunt-bower-task#119]
I don't know if this is a polymer problem or a grunt-bower-task problem, but regular `bower install` works fine.  I add any polymer components to my `bower.json`, such as

``` bash
"dependencies": {
  "core-input": "Polymer/core-input#~0.3.3"
}
```

and then run `grunt bower` and get this:

``` bash
Running "bower:install" (bower) task
Fatal error: Maximum call stack size exceeded
```

**Comment by jorgecasar** [comment]
I'm having the same issue. And with polymer dependencies too.

I've just got this error:

```
FATAL ERROR: JS Allocation failed - process out of memory
Abort trap: 6
```

**Comment by LynxDelli** [comment]
Same issue with Ember & jquery and whatever I tried.

**Comment by topless** [comment]
+1

**Comment by topless** [comment]
It seems to be working on my linux machine but not on the mac.

**Comment by ktilcu** [comment]
@indiesquidge Seems this is resolved in 0.4.0.... Close?

**Comment by indiesquidge** [comment]
@ktilcu, yes! Sorry my response took a while; it seems to be working now, thanks :)

**Comment by ktilcu** [comment]
:+1:

**Comment by theblang** [comment]
Same problem here.  

Using grunt: 0.4.2 and grunt-bower: 0.15.0

Aborted due to warnings.
Running "bower:dev" (bower) task
Warning: Maximum call stack size exceeded
